examples/docs/src/pages/en/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Hello, Documentation!
-layout: ../layouts/Main.astro
```

Comment:
repeating past comment: no more need for this Introduction page, safe to remove and just keep the `getting-started` and `example` page.

Or, if you prefer to keep it: Make this `introduction.md` and put it in the sidebar, right above "Getting Started".